scripts/rc.py

```diff
@@ -190,7 +190,7 @@ def os_main(args):
         exit(1)
 
 ############ Data Processing Functions ######################
-def process_cdqr(cdqr,args):
+def dp(cdqr,args):
```

Comment:
Was this the right function to change the name of?

Reply:
That shouldn't have been changed